Ex10_Answers.py

```diff
@@ -0,0 +1,159 @@
+##Ex 10
+
+## Question 1
+
+#import packages
+
+import pandas
+import scipy
+import scipy.integrate as spint
+from plotnine import *
+
+
+def ddSim (y,t0,r,K): #Define basic model for density-dependent growth
+    N=y[0]
+    dNdt=r*(1-N/K)*N
+    return [dNdt]
+    
+K=100 #Define a set carrying capacity & initial population size
+N0=[10]
+times=range(0,100)
+  
+rs=[-0.1,0.1,0.4,0.8,1.0] #Define a list of growth rates for each individual population
+store_rs=pandas.DataFrame({"time":times,"r1":0,"r2":0,"r3":0,"r4":0,"r5":0}) #Initialize a data frame to store population size, with
+                                                                             #columns labeled for times, and each growth rate
+for i in range(0,len(rs)): #Set up a for loop that runs through each growth rate in the list, and models population growth using
+    params=(rs[i],K)       #established parameters
+    sim=spint.odeint(func=ddSim,y0=N0,t=times,args=params)
+    store_rs.iloc[:,i]=sim[:,0] #Store population values in the column appropriate for the current growth rate being modeled
+
+rates=ggplot(store_rs,aes(x="time",y="N"))+theme_classic() #Set up a base plot for the population curves
+rates=rates+geom_line(store_rs,aes(y="r1")) #Add each population curve to the graph
+rates=rates+geom_line(store_rs,aes(y="r2"))
+rates=rates+geom_line(store_rs,aes(y="r3"))
+rates=rates+geom_line(store_rs,aes(y="r4"))
+rates=rates+geom_line(store_rs,aes(y="r5"))
+
+rates #Display the graph
```

Comment:
Good job! If you want to color code each line, you can use: 
ggplot(store_rs,aes(x="time",y="r1",color="10"))+geom_line(store_rs,aes(y="r2",color="50"))+geom_line(store_rs,aes(y="r3",color="100"))